### DISCUSSION TITLE
Abort Reason

### DISCUSSION THREAD

**Discussion by hfjn** [argoproj/argo-rollouts#3956] [Q&A]
So we have been rolling out ArgoRollouts to a few applications in the last couple of days and overall are super happy with it. But one behaviour that has caused a lot of confusion is the reason for some of it's actions. 

E.g. last night we got a notification that a rollout was rolled back. The rollout is now in the degraded state and marked as aborted. But all analyses are successful. 
We have `progressDeadlineAbort` set to true so this is likely the reason for it, but there could also be other reasons for it. Is there any place that I don't know of where these reasons are logged so it easier for a user to understand what is going on?

**Comment by kostis-codefresh** [comment]
How much time do you have a Canary/Blue/Green deployment running?

The controller logs are obviously the best place to see what is going on.
